pytoport/__init__.py

```diff
@@ -137,15 +164,19 @@ def get_minimum(data):
 
     return "%s+ # %s" % (ver, ", ".join(others))
 
+def gen_dep(pkg):
+    ports = FreeBSD_ports()
+    portpath = ports.find_portdir("py36-" + pkg)
```

Comment:
is py36 here right?

Reply:
No. This probably needs something to find the correct version of python but for now I'll just change it and hope to do that in a latter commit.